drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -202,7 +202,7 @@ function DragAndDropTemplates(configuration) {
                 h('div.modal-window-overlay'),
                 h('div.modal-window', {attributes: {role: 'dialog', 'aria-labelledby': 'modal-window-title'}}, [
                     h('div.modal-header', [
-                        h('h2.modal-window-title', gettext('Keyboard Help'))
+                        h('h2.modal-window-title#modal-window-title', gettext('Keyboard Help'))
```

Comment:
This caused a non-unique ID when 2 problems are on the same page. It is very easy to look for these sort of issues using Chrome and the aXe developer tool plugin.

![image](https://cloud.githubusercontent.com/assets/484484/17351394/0ebbba34-58fc-11e6-9889-59a526f64f59.png)